modules/microshift-install-system-requirements.adoc

```diff
@@ -8,15 +8,24 @@
 
 The following conditions must be met prior to installing {microshift-short}:
 
-* A compatible version of {op-system-base} or {op-system-ostree}.
+* A compatible version of {op-system-base-full}. For more information, see the "Compatibility table" section.
 * AArch64 or x86_64 system architecture.
 * 2 CPU cores.
-* 2 GB RAM for {microshift-short} or 3 GB RAM, required by {op-system-base} for networked-based HTTPs or FTP installations.
+* 2 GB RAM. Installing from the network (UEFI HTTPs or PXE boot) requires 3 GB RAM for {op-system-base}.
 * 10 GB of storage.
 * You have an active {microshift-short} subscription on your Red Hat account. If you do not have a subscription, contact your sales representative for more information.
-* You have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient capacity for the Persistent Volumes (PVs) of your workload.
+* If your workload requires Persistent Volumes (PVs), you have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient free capacity for the workloads.
 
 [IMPORTANT]
 ====
-Secure access must be configured before running {microshift-short} to ensure proper functionality and security. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/assembly_using-secure-communications-between-two-systems-with-openssh_securing-networks[Using secure communications between two systems with OpenSSH].
+These requirements are the minimum system requirements for {microshift-short} and {op-system-base-full}. Add the system requirements for the workload you plan to run.
+
+For example, if an IoT gateway solution requires 4 GB of RAM, your system needs to have at least 2GB for {op-system-base-full} and {microshift-short}, plus 4GB for the workloads. 6GB of RAM is required in total.
```

Comment:
Not enough to hold up a merge review, but be mindful of how you write sizes. You have "4 GB" with a space, and "2GB, 4GB, 6GB" with no space. Correct way is with a space. https://www.ibm.com/docs/en/ibm-style?topic=measurement-units#general-guidelines